l10n_it_fatturapa/i18n/es.po

```diff
@@ -3,68 +3,57 @@
 # * l10n_it_fatturapa
 #
 # Translators:
-# Aeshna <synieoka@gmail.com>, 2014
-# Albert Cervera i Areny <albert@nan-tic.com>, 2009
-# Andrii Dvoiak <andrii@preply.com>, 2015
-# danimaribeiro <danimaribeiro@gmail.com>, 2015
-# Dmytro Voloshyn <dmytro@preply.com>, 2014-2016
-# Dorin Hongu <dhongu@gmail.com>, 2015
-# Hotellook, 2014
-# iNastya <nastya@preply.com>, 2015
-# inken <inken.bornholdt@techstars.com>, 2015
-# Julia Boyun <yuliaboyun@gmail.com>, 2015
-# Karolina Starzewska <kstarzewska@gmail.com>, 2015
-# Katharina Zajac <Kaaaasiaz@yahoo.de>, 2015
-# Kirill Bigai <kirill@preply.com>, 2013
-# Konrad Lubinski <konrad@preply.com>, 2015-2016
-# Lizaveta.Makratsova <lizaveta.makratsova@gmail.com>, 2014
-# Maria Kuliy <maria@preply.com>, 2016
-# Massimiliano Casa <mcasa@cantiericasa.com>, 2015
-# Matjaž Mozetič <m.mozetic@matmoz.si>, 2015
-# oihane <oihanecruce@gmail.com>, 2015
-# Oleksandra <inactive+Oleksandra@transifex.com>, 2014
-# Olga Podgayna <olga@preply.org>, 2015
-# Paolo Valier, 2016
-# Pedro M. Baeza <pedro.baeza@gmail.com>, 2015
-# Serge <serge@preply.com>, 2013,2015
-# Александра Маломуж <omalomug@gmail.com>, 2016
+# OCA Transbot <transbot@odoo-community.org>, 2017
```

Comment:
Non bisognerebbe modificare i file .po direttamente, in quanto si potrebbero creare conflitti con weblate